roles/nova-common/tasks/main.yml

```diff
@@ -74,6 +74,60 @@
     mode: 0640
   when: inventory_hostname in groups['controller']
 
+- name: set uwsgi path (source install)
```

Comment:
Is nova-placement-api service to be install on controllers and computes? if is just controllers than this all probably should move to nova-control role.